_source/api/logzio-public-api.yml

```diff
@@ -2701,6 +2705,50 @@ paths:
         - lang: cURL
           source: 'curl -X POST "https://api.logz.io/v1/shared-tokens" -H  "accept: application/json" -H  "content-type: application/json" -H  "X-API-TOKEN: <token>" -d "{ \"tokenName\": \"Support shared token\", \"filters\":[339]}"'
 
+  # ::::: API TOKENS
+  /v1/api-tokens/sub-account:
+    post: 
+      operationId: createSubaccountAPIToken
+      summary: Create a subaccount API token
+      description: >-
+        Creates a new API token for a subaccount.
+
+      tags:
+        - Manage API tokens
+      parameters: 
+        - in: body
+          name: body
+          required: true  
+          schema:
+            $ref: '#/definitions/CreateSubaccountAPITokenRequest'
+      responses:
+        200:
+          description: successful operation
+          schema:
+            $ref: '#/definitions/SubaccountAPIToken'
```

Comment:
if this is a response object then it should be `ApiTokenResponse`

Reply:
Done

Reply:
i just had a discussion on Thursday with @eranshahar and @yuvalschneider to change the response so that it excludes the id. If we go this way it means we change the response object. Also, it will mean removing the description of the ApiTokenResponse **id** field i sent @YajB via slack.

Reply:
@Jonathan-Eng @yuvalschneider @eranshahar - 
When there is a decision, I will need explicit explanations of what needs to change.